src/install.js

```diff
@@ -3,19 +3,21 @@ import { Ability, subject } from '@casl/ability';
 
 import * as components from './components';
 
-const install = (Vue, {prefix} = {}) => {
-  for (let key in components) {
-    let _key = prefix ? prefix + key : key;
-    Vue.component(_key, components[key]);
+const install = (app, {prefix, loadComponents = true} = {}) => {
+  if (loadComponents) {
+    for (let key in components) {
+      let _key = prefix ? prefix + key : key;
+      app.component(_key, components[key]);
+    }
   }
 
-  Vue.use(abilitiesPlugin, new Ability);
+  app.use(abilitiesPlugin, new Ability);
   // Vue.prototype.$createEntity = function (path, item) {
   //   const classFn =  new Function('item', `return new class ${path} {constructor(args) {Object.assign(this, item)}}`);
   //   const newClass = classFn(item);
   //   return newClass;
   // };
-  Vue.prototype.$subject = subject;
+  app.prototype.$subject = subject;
```

Comment:
this needs a provide and globalProperties. `app.config.globalProperties.$subject = subject;
  app.provide('$subject', subject);`

Reply:
I see. Thanks for taking care of this.